src/backend/helpers.js

```diff
@@ -60,16 +60,17 @@ const tagType = {
 };
 
 function mdToHtml(text) {
-    // text = text.replace(/([,;/])(?=\S)/g, '$1\u200b');
-    let result = converter.makeHtml(text).replace(/\n/g, '<br>');
+    if (text === undefined)
+        return undefined;
 
-    // if (result.startsWith('<p>'))
-    //     result = result.substr(3, result.length - 7);
-
-    return result;
+    const html = converter.makeHtml(text) || text;

```

Comment:
the `|| text` part seems a bit security-unwise to me. Is this mdToHtml method publicly accessible in any manner?

Reply:
Nope, all this code is server sided.